lib/akami/wsse.rb

```diff
@@ -136,7 +135,7 @@ def wsse_signature
       # First key/value is tag/hash
       tag, hash = signature_hash.shift
 
-      security_hash nil, tag, hash, signature_hash
+      security_hash nil, tag, hash, signature_hash, true
```

Comment:
Maybe out of scope here but I think with so many args this would benefit from keywork args for clarity